asapdiscovery-docking/asapdiscovery/docking/openeye.py

```diff
@@ -55,6 +55,9 @@ class POSITDocker(DockingBase):
         POSIT_METHOD.ALL, description="POSIT method to use"
     )
     use_omega: bool = Field(True, description="Use omega to generate conformers")
+    omega_dense: bool = Field(
```

Comment:
Should be added to the CLI?

Reply:
For large scale CI I don't think we should have as an option, but for small scale and cross docking I will add it.

Reply:
EDIT: small scale isn't in yet so just cross docking.